client/task-list/tasks/payments/PaymentGatewaySuggestions/components/List/test/list.js

```diff
@@ -0,0 +1,81 @@
+/**
+ * External dependencies
+ */
+import { render } from '@testing-library/react';
+
+/**
+ * Internal dependencies
+ */
+import { List } from '../List';
+
+jest.mock( '@woocommerce/tracks', () => ( {
+	recordEvent: jest.fn(),
+} ) );
+
+jest.mock( '@wordpress/data', () => {
+	// Require the original module to not be mocked...
+	const originalModule = jest.requireActual( '@wordpress/data' );
+
+	return {
+		__esModule: true, // Use it when dealing with esModules
+		...originalModule,
+		useSelect: jest.fn().mockReturnValue( {
+			menuItems: [],
+		} ),
+	};
+} );
+
+const mockGateway = {
+	id: 'mock-gateway',
+	title: 'Mock Gateway',
+	plugins: [],
+	postInstallScripts: [],
+	requiredSettings: [],
+	installed: false,
+	needsSetup: false,
+	enabled: false,
+};
+
+const defaultProps = {
+	heading: 'Test heading',
+	markConfigured: jest.fn(),
+	recommendation: 'testId',
+	paymentGateways: [ mockGateway ],
+};
+
+describe( 'PaymentGatewaySuggestions > List', () => {
+	it( 'should display correct heading', () => {
+		const { queryByText } = render( <List { ...defaultProps } /> );
+
+		expect( queryByText( defaultProps.heading ) ).toBeInTheDocument();
+	} );
+
+	it( 'should display gateway title', () => {
+		const { queryByText } = render( <List { ...defaultProps } /> );
+
+		expect( queryByText( mockGateway.title ) ).toBeInTheDocument();
+	} );
+
+	it( 'should display the "Enable" button when setup is NOT required', () => {
```

Comment:
Nice tests!  Not sure if this is fully ready for review or not, but would be great to get some tests around the "Manage" button as well.

Reply:
Yup! Left this mid-stream last week, but I've added more tests now 👍🏻